script/RumpelConfig.sol

```diff
@@ -76,6 +83,9 @@ library RumpelConfig {
     address public constant MAINNET_METH = 0xd5F7838F5C461fefF7FE49ea5ebaF7728bB0ADfa;
     address public constant MAINNET_RETH = 0xae78736Cd615f374D3085123A210448E74Fc6393;
     address public constant MAINNET_CBETH = 0xBe9895146f7AF43049ca1c1AE358B0541Ea49704;
+    address public constant MAINNET_FXS = 0x3432B6A60D23Ca0dFCa7761B7ab56459D9C964D0;
+    address public constant MAINNET_TBTC = 0x18084fbA666a33d37592fA2633fD49a74DD93a88; // tBTC (v2)
```

Comment:
just to be aware, for whoever is going to resolve merges, both this PR and dorde's (#38 ) add TBTC

Reply:
cc @mijovic